docs/conversation.md

```diff
@@ -88,6 +88,7 @@
 | `statusMessage`         | string  | v4    |         | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set                                                                                                                                                                                                                                                                                                       |
 | `participants`          | array   | v1    | v2      | **Removed**                                                                                                                                                                                                                                                                                                                                                                                       |
 | `guestList`             | string  | v1    | v2      | **Removed**                                                                                                                                                                                                                                                                                                                                                                                       |
+| `avatarUrl`             | string  | v4    |         | Avatar URL of the conversation including a version flag `v=…` for easier expiration of the avatar in case a moderator updates it, since the avatar endpoint should be cached for 24 hours.                                                                                                                                                                                                        |
```

Comment:
I think this parameter is a bit problematic. It will make the avatar look updated on the conversation list, but e.g. search results, notifications, etc still show the old one. So basically we would need to tell clients to track this value and when it is different, they should invalidate their avatar image cache.

Reply:
cc @Ivansss @mahibi 

Any idea what the best way would be? I'd like to have it similar to user avatars. They work without any additional information, you just need the userid (conversation token) and can make your URL. It's cached so can take some time until others see it. Should be fine, unless you did the change yourself.
Would you think it's okay to have it cached/old on the mobile apps when you changed it in the web? Or what would be a good way to handle this?

Reply:
> The frontend every time get the version directly from userconfig:

This is only for your very own avatar. It is not done for the avatar of others, see:
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcAvatar/NcAvatar.vue#L627-L647

But that is exactly the problem here. It's basically always the avatar of others 😬

Reply:
I think that this wont be a problem to Talk.

I already implemented to send the version of avatar in Room object, then, in all places that we need to display the current room avatar, will be possible.

In the places that we haven't the Room object, we will send the URL of avatar and we already have a contract to do this, have places in server that need a property called icon, other use iconUrl and we can do the same in Talk.

Reply:
That is exactly why I started the discussion here. We will NOT be able to add the avatar URL on all things. But anyway.

We can move this to a follow up issue so we can merge this PR